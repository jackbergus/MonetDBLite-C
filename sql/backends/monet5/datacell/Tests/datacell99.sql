#set up the minimal test environment for datacell

drop procedure datacell.transport;
drop table datacell.X;
drop table datacell.Y;
