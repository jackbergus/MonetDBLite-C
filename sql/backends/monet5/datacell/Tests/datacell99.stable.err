stderr of test 'datacell99` in directory 'sql/backends/monet5/datacell` itself:


# 20:48:37 >  
# 20:48:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31907" "--set" "mapi_usock=/var/tmp/mtest-7013/.s.monetdb.31907" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/current//Linux/var/MonetDB/mTests_sql_backends_monet5_datacell" "--set" "mal_listing=0"
# 20:48:37 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/current//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 31907
# cmdline opt 	mapi_usock = /var/tmp/mtest-7013/.s.monetdb.31907
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/current//Linux/var/MonetDB/mTests_sql_backends_monet5_datacell
# cmdline opt 	mal_listing = 0
# cmdline opt 	gdk_debug = 536870922

# 20:48:37 >  
# 20:48:37 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-7013" "--port=31907"
# 20:48:37 >  


# 20:48:37 >  
# 20:48:37 >  "Done."
# 20:48:37 >  

