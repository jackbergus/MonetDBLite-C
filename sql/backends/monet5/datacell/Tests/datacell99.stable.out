stdout of test 'datacell99` in directory 'sql/backends/monet5/datacell` itself:


# 20:48:37 >  
# 20:48:37 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31907" "--set" "mapi_usock=/var/tmp/mtest-7013/.s.monetdb.31907" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/current//Linux/var/MonetDB/mTests_sql_backends_monet5_datacell" "--set" "mal_listing=0"
# 20:48:37 >  

# MonetDB 5 server v11.18.0
# This is an unreleased version
# Serving database 'mTests_sql_backends_monet5_datacell', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.591 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2014 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.ins.cwi.nl:31907/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-7013/.s.monetdb.31907
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded
# MonetDB/DataCell loaded

Ready.

# 20:48:37 >  
# 20:48:37 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-7013" "--port=31907"
# 20:48:37 >  

#drop procedure datacell.transport;
#drop table datacell.X;
#drop table datacell.Y;

# 20:48:37 >  
# 20:48:37 >  "Done."
# 20:48:37 >  

